tests/inject/dynamic/image-analysis.tests.ts

```diff
@@ -40,7 +40,7 @@ function getSVGImageCSS(svg: string, width: number, height: number, selector: st
     );
 }
 
-export const images = {
+const images = {
```

Comment:
This was causing an error at some point, and I could not recreate it. But the images constant does not need to be exported from this file, so I removed it.